docs/source/installed_packages.rst

```diff
@@ -0,0 +1,20 @@
+Using Installed Packages
+========================
+
+
+Making PEP 561 compatible packages
+**********************************
+
+Packages that supply type information should put a ``py.typed``.
```

Comment:
... in their package directory. Maybe we should add an example of how to do this with `setup.py`?

Reply:
Yes an example in the docs seems like a good idea, I will add that.